heartbeat/Route

```diff
@@ -227,6 +227,15 @@ route_stop() {
 }
 
 route_status() {
+    if [ -n "${OCF_RESKEY_device}" ]; then
+	# Must check if device exists or is gone.
+	# If device is gone, route is also unconfigured.
+	device_exists="$(ip link show dev ${OCF_RESKEY_device})"
```

Comment:
I just realised we dont use device_exist, so you can just run the command here wiithout assignment.

Reply:
Indeed, I've dropped the assignment and redirect the output of the command (both `stdout` and `stderr`) to `/dev/null` now since we don't use either. Thanks!